Docs/source/pages/advanced-usage.md

```diff
@@ -484,6 +484,35 @@ Use [Reverse Port Forwarding](#what-is-reverse-port-forwarding-and-when-to-use-i
     On mobile devices, AltDriver can interact only with a single app at a time and the app needs to be in focus. In case of 2 drivers and 2 apps, you need to switch (in your test scripts) between the applications. This is due to the fact that on Android/iOS only one application is in focus at a time, even when using split screen mode.
 ```
 
+## Execute tests concurrently
+
+In the `AltDriver` constructor you have the option to specify multiple tags. The available tags are: app name, platform, platform version, device instance id and app id. The app id can be used to uniquely identify an app. In case you specify no tags, the tests will be run on a randomly chosen app.
+
+Keep in mind that, the tags given in the constructor will choose one random free app satisfying the requirements. Only one test can run on one app simultaneously. If you want to run the same tests on multiple apps concurrently you have to use the `dotnet test` command as many times as many apps you want your tests to be executed on.
```

Comment:
I would say:

`If you want to run the same tests on multiple apps concurrently you have to start the `dotnet test` command multiple times, once for each app/device that you want your tests to be executed on.`

Reply:
we also should say `dotnet test` or `pytest` or whichever command you use to start your tests.

Reply:
Maybe we should specify that the `dotnet test` command should be executed in different terminals?

Reply:
👍